components/org.wso2.carbon.identity.handler.event.account.lock/src/main/java/org/wso2/carbon/identity/handler/event/account/lock/internal/AccountServiceComponent.java

```diff
@@ -52,6 +55,17 @@ protected void activate(ComponentContext context) {
         if (log.isDebugEnabled()) {
             log.debug("AccountDisableHandler is registered");
         }
+        XACMLEndpointConfigImpl xacmlEndpointConfigImpl = new XACMLEndpointConfigImpl();
```

Comment:
Couldn't add the XACML endpoint in XACML because of the cyclic dependency since XACML endpoint needs carbon identity framework as a dependency.